packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -155,13 +155,19 @@ const isCampaignEvent = (event: Event) => {
 
 export const shouldTrackHistoryPageView = (
   trackingOption: Options['trackHistoryChanges'],
-  newURL: string,
-  oldURL: string,
+  newURLStr: string,
+  oldURLStr: string,
 ): boolean => {
   switch (trackingOption) {
-    case 'pathOnly':
-      return newURL.split('?')[0] !== oldURL.split('?')[0];
+    case 'pathOnly': {
+      if (oldURLStr == '') return true;
+      const newURL = new URL(newURLStr);
+      const oldURL = new URL(oldURLStr);
+      const newBaseStr = newURL.protocol + '//' + newURL.host + newURL.pathname;
+      const oldBaseStr = oldURL.protocol + '//' + oldURL.host + oldURL.pathname;
```

Comment:
origin contains protocol + host https://web.dev/articles/url-parts#url_analyzer
```suggestion
      const newBaseStr = newURL.origin + newURL.pathname;
      const oldBaseStr = oldURL.origin + oldURL.pathname;
```

Reply:
Updated.